package.json

```diff
@@ -52,33 +52,37 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-webpack-plugin": "^3.1.1",
     "fork-ts-checker-webpack-plugin": "^7.2.0",
-    "glob": "^8.0.3",
+    "glob": "^10.0.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "27.5.0",
+    "jest": "^29.3.1",
     "lodash": "latest",
     "moment": "2.29.4",
     "prettier": "^2.5.0",
     "replace-in-file-webpack-plugin": "^1.0.6",
     "sass": "1.56.1",
-    "sass-loader": "13.2.1",
-    "style-loader": "3.3.2",
+    "sass-loader": "13.2.0",
+    "style-loader": "3.3.1",
     "swc-loader": "^0.2.3",
     "ts-node": "^10.5.0",
     "tsconfig-paths": "^4.1.0",
     "tslib": "2.3.1",
     "typescript": "^4.4.0",
     "webpack": "^5.69.1",
     "webpack-cli": "^4.9.2",
-    "webpack-livereload-plugin": "^3.0.2"
+    "webpack-livereload-plugin": "^3.0.2",
+    "@types/react": "17.0.42",
+    "@types/react-dom": "17.0.14",
+    "jest-environment-jsdom": "^29.3.1"
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "9.1.2",
-    "@grafana/runtime": "9.1.2",
-    "@grafana/ui": "9.1.2",
+    "@grafana/data": "9.5.1",
```

Comment:
Do we need to bump these and if so should we update the grafanaDependency in plugin.json?

Reply:
Okie dokie, let's do like this. 🚀